L1Trigger/TrackFindingTracklet/src/TrackletCalculatorBase.cc

```diff
@@ -209,16 +213,16 @@ void TrackletCalculatorBase::exactprojdisk(double zproj,
 void TrackletCalculatorBase::addDiskProj(Tracklet* tracklet, int disk) {
   FPGAWord fpgar = tracklet->fpgarprojdisk(disk);
 
-  if (fpgar.value() * settings_->krprojshiftdisk() < 12.0)
+  if (fpgar.value() * settings_.krprojshiftdisk() < 12.0)
```

Comment:
still a few magic numbers here and below

Reply:
fixed